narwhal/worker/src/handlers.rs

```diff
@@ -147,8 +149,13 @@ impl<V: TransactionValidator> PrimaryToWorker for PrimaryReceiverHandler<V> {
         &self,
         request: anemo::Request<WorkerSynchronizeMessage>,
     ) -> Result<anemo::Response<()>, anemo::rpc::Status> {
+        let Some(network) = self.network.as_ref() else {
```

Comment:
Whoops missed this on review, good catch

Reply:
This is probably a good indication that we need tests with both flavours (with worker local & remote as well)